src/language/schema/ast.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *  Copyright (c) 2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+import {
```

Comment:
For similar clarity reasons, I would suggest duplicating these few rules here instead of trying to isolate them.